alot/account.py

```diff
@@ -214,6 +214,9 @@ def errb(failure):
             logging.error(failure.value.stderr)
             raise SendingMailFailed(errmsg)
 
+        # make sure self.mail is a string
+        mail = str(mail)
```

Comment:
This is going to fail spectacularly if mail is a unicode object that contains a non-ascii encodable character.